orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -416,6 +416,10 @@ TEST(GradientCheckerTest, LogGrad) {
   EXPECT_IS_TINIER_THAN(max_error, error_tolerance);
 }
 
+TEST(GradientCheckerTest, ExpGrad) {
+  UnaryOpGradientTest("Neg");
```

Comment:
"Exp"

Reply:
Ha, thanks for catching that! Seemed too easy :P.